src/gunicorn_prometheus_exporter/plugin.py

```
@@ -6,19 +6,24 @@
 """
 
 import logging
 import os
 import time
 
 import psutil
+from gunicorn.arbiter import Arbiter
 from gunicorn.workers.sync import SyncWorker
 
 from .metrics import (
+    MASTER_WORKER_RESTARTS,
```

Ruff (0.8.2)

`.metrics.MASTER_WORKER_RESTARTS` imported but unused

Remove unused import: `.metrics.MASTER_WORKER_RESTARTS`

(F401)